src/GlobalTarget.js

```diff
@@ -64,6 +64,8 @@ class GlobalTarget extends Component {
           stack: stack.filter(({ type }) => type !== jsx.type),
         }),
         () => {
+          this.props.onRemoveStackElement(jsx)
```

Comment:
Fails when onRemoveStackElement is `null` or `onRemoveStackElement` is not a function should we do such a check? onRemoveStackElement && typeof onRemoveStackElement === 'function' && onRemoveStackElement(...)

Reply:
it is not being done for the other handlers, as well and it has a default value. I wanted to stay consistent with the existing code.

Reply:
yeah true :D tried to push improvement on all ;)